### DISCUSSION TITLE
LYSO scintillator

### DISCUSSION THREAD

**Discussion by altair259** [OpenGammaProject/Open-Gamma-Detector#27] [Q&A] [answered]
Hello,
Are LYSO scintillator as replacement for NaI?

**Comment by NuclearPhoenixx** [comment, marked as answer]
Hi, yes you can use LYSO scintillators instead of NaI! They're actually great crystals (faster decay time, double the density, ...), but keep in mind that due to the material composition you will always have an ambient Lu-176 background. You will have to subtract that background for your other measurements.

If you get one of the smaller ones like 4x4 mm is pretty much the ideal size for the SiPM coupling. Though, you will have a worse detection efficiency at higher energies then (compared to a larger scintillator).
